shared/torngit/github.py

```diff
@@ -1285,7 +1285,7 @@ async def get_repos_from_nodeids_generator(
                         ),
                         "private": raw_repo_data["isPrivate"],
                         "branch": (
-                            default_branch.get("name") if default_branch else None
+                            default_branch.get("name") if default_branch else "master"
```

Comment:
Wouldn't we want to do main?

Reply:
other places in this code do "master" and i thought i should be consistent